source/Ship.cpp

```diff
@@ -601,6 +625,20 @@ void Ship::Load(const DataNode &node)
 // loaded yet. So, wait until everything has been loaded, then call this.
 void Ship::FinishLoading(bool isNewInstance)
 {
+	// Automatically create an explosion for this ship if requested but
+	// only if the ship does not already have a defined explosion.
```

Comment:
Are we sure that we to have logic for explosion hierarchy? I'm not sure if there's a use-case for overriding the automatic explosion, so just directly setting the explosion in `Load` would be nicer and make it so that not every ship needs to carry around a vestigial `autoExplosion` variable that's never used again.

Reply:
@Amazinite requested so.